### PR TITLE
[PREVIEW] SL-1940 try to fix tests

### PR DESCRIPTION
**Pull request by arompa**
Fingers crossed

### CODE REVIEW

**Comment by arompa** [comment]
build

**Comment by arompa** [comment]
Green on first try :)

File: e2e/specs/problem-list.e2e-spec.ts [hunk 1]
```diff
@@ -54,11 +54,12 @@ describe('Problem list tests', () => {
             await navigationFlow.goToProblemsPage();
         });
 
-        afterEach(async () => {
```
Comment (zielinskikamil) [inline e2e/specs/problem-list.e2e-spec.ts]: is this needed? I think that the new reported includes error msgs.

Comment (arompa) [reply e2e/specs/problem-list.e2e-spec.ts]: I honestly don't know :) Didn't want to just throw it away though

**Pull request merged**